### ISSUE TITLE
Broken link in description

### ISSUE THREAD

**Issue by jsejcksn** [lisezmoi/alpha-bet#7]
This seems cool, but the link ([http://lisezmoi.2015.nodeknockout.com/](http://lisezmoi.2015.nodeknockout.com/)) in the description is broken. 🙁

**Comment by bpierre** [comment]
Indeed, thanks for letting us know. We will host it somewhere else and update this issue once it’s live again.

**Comment by jsejcksn** [comment]
Thanks!

(Also, unrelated: I sent a message to the lisezmoi contact email about scri.ch because it wasn't an "issue", so be on the lookout for it!)

**Comment by jsejcksn** [comment]
Did you change your mind about this? Will you update, please?
